metaci/release/models.py

```diff
@@ -122,7 +122,16 @@ def update_from_github(self):
         update_release_from_github(self)
 
     def save(self, *args, **kw):
+        if not self.version_number:
+            self.version_number = self.repo.latest_release.git_tag.strip(
+                f"{self.repo.release_tag_regex}"
+            )
+        if not self.version_name:
+            self.version_name = self.repo.latest_release.git_tag.strip(
+                f"{self.repo.release_tag_regex}"
+            )
```

Comment:
What are you trying to do here? It looks like if a Release is saved that does not yet have a version name or number, they will get filled in by copying from the latest existing release. That doesn't sound right (we don't want to create a new release with the same number as one that already exists).